### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by kazim110**
1. Create vets, specialisations and visits tables
2. Insert data to these tables
3. Doing some queries from these tables

There are screenshots of those queries:

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -83,4 +83,63 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
...
+JOIN animals a ON v.animal_id = a.id 
+GROUP BY a.name 
+ORDER BY visit_count DESC LIMIT 1;
+
+SELECT vet.name AS vet_name, MIN(v.date) AS first_visit_date 
+FROM visits v 
+JOIN vets vet ON v.vet_id = vet.id 
+JOIN animals a ON v.animal_id = a.id 
+WHERE vet.name = 'Maisy Smith' 
+GROUP BY vet.name;
```
Comment (mikethreels) [inline queries.sql]: - [ ] In this query you are returning the vet name however according to the requirements you're supposed to return the animal name who visited Maisy last

File: queries.sql [hunk 2]
```diff
@@ -83,4 +83,63 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
...
+WHERE s.vet_id IS NULL;
+
+SELECT s.name AS specialty, COUNT(*) AS visit_count 
+FROM visits v 
+JOIN animals a ON v.animal_id = a.id 
+JOIN specializations sp ON a.species_id = sp.species_id 
+JOIN species s ON sp.species_id = s.id 
+WHERE a.owners_id = (SELECT id FROM owners WHERE full_name = 'Maisy Smith') 
+GROUP BY s.name 
+ORDER BY visit_count DESC LIMIT 1;
```
Comment (mikethreels) [inline queries.sql]: - [ ] This query doesn't return anything at the moment, please review it, if you need extra help please leave a message in the PR.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kazim, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the queries.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kazim110** [comment]
These are screenshots of last changes:

**Review by DeliceLydia — APPROVED** [review]
## Hi @kazim110, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**